docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+
+```bash
+npx testcafe
+```
+
+### Step 5. Configure the reporter
+
+Turn on the [JUnit](https://plugins.jenkins.io/junit/) reporter plugin: add the "Publish JUnit test result report" *post-build action*. To display screenshots and videos alongside your test results, select the option "Include links to TestCafe artifacts" from the "Additional test report features" drop-down.
+
+Important: Don't forget to check the "Retain long standard output/error" box. Otherwise the plugin will automatically truncate information concerning test screenshots and videos.
+
+![JUnit reporter configuration screen](/testcafe/images/jenkins-plugin/junit-reporter-configuration.png)
+
+### Step 6. Run the build
+
+Save the changes to your project and press the "Build now" button on the project page:
+
+![Jenkins Project menu with the Build Now Button highlighted](/testcafe/images/jenkins-plugin/build-now-button.png)
+
+## Review the results
+
+After the tests are completed, press the following link on the build page to view the results:
+
+![Test results link](/testcafe/images/jenkins-plugin/test-results-link.png)
+
+There you'll see links to screenshots and videos taken during the test.
+
+![Test results view](/testcafe/images/jenkins-plugin/test-results-view.png)
+
+These artifacts instantly indicate if the Cart button is present on the page.
+
+## Jenkins Pipeline integration
+
+If you don't want to use the Jenkins GUI to launch these tests, you can always make use of [Jenkins Pipeline](https://www.jenkins.io/doc/book/pipeline/). It lets you declaratively configure your CI/CD setup via a Jenkinsfile, a text file checked into the project's source control repository. Here's a sample Jenkinsfile to be used with TestCafe:
+
+```java
+pipeline {
+    agent any
+
+    stages {
+        stage('Run E2E tests') {
+            steps {
+                sh 'npm install'
+                sh 'npm run test-e2e'
+            }
+            post {
+                always {
+                          junit keepLongStdio: true,
+                          testDataPublishers: [[$class: 'TestCafePublisher']],
+                          testResults: '*.xml'
+                }
+            }
+        }
+    }
+}
+```
+
+Note: The `test-e2e` npm script in this example is responsible for launching TestCafe. See the [TestCafe CLI](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html) documentation for more information on the subject, and don't forget to manually set the reporter option as outlined earlier in this article.
+
+--
+
+If you enjoyed this guide, you might like our other articles on the subject of debugging: [How to Debug Tests in TestCafe: Quick Guide](https://devexpress.github.io/testcafe/media/team-blog/how-to-debug-tests-in-testcafe-quick-guide.html) and [How to speed up debugging in TestCafe: Tips and Tricks](https://devexpress.github.io/testcafe/media/team-blog/how-to-speed-up-debugging-in-testcafe-tips-and-tricks.html).
```

Comment:
```suggestion
If you enjoyed this guide, you may like our other articles on debugging: [How to Debug Tests in TestCafe: Quick Guide](https://devexpress.github.io/testcafe/media/team-blog/how-to-debug-tests-in-testcafe-quick-guide.html) and [How to speed up debugging in TestCafe: Tips and Tricks](https://devexpress.github.io/testcafe/media/team-blog/how-to-speed-up-debugging-in-testcafe-tips-and-tricks.html).
```

Reply:
Can I substitute `help topics` for `guides`? These aren't really help topics in the proper sense, as they aren't a part of a knowledge base. This is what I imagine when I hear the words 'help topic":
![image](https://user-images.githubusercontent.com/43554315/88168148-abb5e380-cc22-11ea-9d87-463fef43a262.png)

Reply:
As far as I know, "help topic" is the phrase that Ray prefers, so I think we should follow his guidelines.